src/main.c

```diff
@@ -252,8 +252,8 @@ gboolean vb_load_uri(Client *c, const Arg *arg)
         rp  = realpath(path, NULL);
         uri = g_strconcat("file://", rp, NULL);
         free(rp);
-    } else if (strchr(path, ' ') || !strchr(path, '.')) {
-        /* use a shortcut if path contains spaces or no dot */
+    } else if (!strchr(path, '.')) {
```

Comment:
You are totally right, my bad. I misinterpreted the condition. I will update the commit.